CO2_Gadget_EINK.h

```diff
@@ -177,7 +179,7 @@ void setElementLocations() {
     uint16_t tbw, tbh;
 
     // Common positions for most displays
-    elementPosition.co2X = 2;       // Left screen
+    elementPosition.co2X = 0;       // Left screen
```

Comment:
So, you don't like the rectangle. 🤔
I think it helps structure the display. Let's discuss it with the Alpha Testers goup!

Reply:
Rectangle is also displayed... I don't like, but It's still displayed. co2X is used to position box. Text is center in box. No problem to show rectangle added to text.